File: 05_arrays/05_04_remove_duplicates_from_sorted_array/remove_duplicates_from_sorted_array.js
const removeDuplicates = (arr = []) => {
    let j = 1;
    for (let index = 1; index < arr.length; index++) {
        if (arr[index]  != arr[index-1]) {
            arr[j++] = arr[index];
        }
    }
    return j;
}

const elements = [0, 0, 1, 1, 1, 2, 2, 3, 3, 4];
console.log("Elements:-");
console.log(elements);
const nonDuplicates = removeDuplicates(elements);
console.log(`NonDuplicates: ${nonDuplicates}`);
for (let index = 0; index < nonDuplicates; index++) {
    process.stdout.write(elements[index] + " ");
}

File: 05_arrays/05_04_remove_duplicates_from_sorted_array/remove_duplicates_from_sorted_array.py
def removeDuplicates(nums: list) -> int:
    j = 1
    for i in range(1, len(nums)):
        if nums[i] != nums[i - 1]:
            nums[j] = nums[i]
            j += 1
    return j


elements = [0, 0, 1, 1, 1, 2, 2, 3, 3, 4]
print("Elements:-")
print(elements)
nonDuplicates = removeDuplicates(elements)
print(f"NonDuplicates: {nonDuplicates}")
for i in range(nonDuplicates):
    print(elements[i], end=" ")